demo/vl-map-search.html

```diff
@@ -13,6 +13,12 @@
   <link rel="stylesheet" type="text/css" href="/node_modules/vl-ui-body/dist/style.css" />
   <link rel="stylesheet" type="text/css" href="/node_modules/vl-ui-form-message/dist/style.css" />
   <link rel="stylesheet" type="text/css" href="/node_modules/vl-ui-button/dist/style.css" />
+
+  <style>
+    vl-map-search {
+        --vl-search-width: 100%;
+    }
+  </style>
```

Comment:
Het zoeken bij 'Handmatige koppeling afzonderlijke adres zoek functionaliteit' werkt niet op de demo pagina. De kaart zoomt niet in naar de plaats waar gezocht wordt.

Reply:
Het werkt wel... was vergeten op de knop 'Koppel zoekbalk aan kaart' te drukken.